Dependencies/WebSocket.java

```diff
@@ -0,0 +1,39 @@
+// add package
```

Comment:
what's this comment mean?

Reply:
it's just for the user to add whatever package they choose to put this file in, it's probably redundant, I'll get rid of it.